app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String, unique=True)
```

Comment:
I think we should allow for [zip+4](https://en.wikipedia.org/wiki/ZIP_code#ZIP.2B4)